universal-login-commons/lib/contracts/FIFSRegistrar.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Move to test directory

Reply:
Moved helpers to utils, so it is not necessary already.